GitExtensions.VS2015.sln

```diff
@@ -0,0 +1,563 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
:question: Why put this in a separate solution file? Only two solution files are necessary: one for Windows, and one for Mono.